bridge-fungible-token/tests/utils/environment.rs

```diff
@@ -189,15 +190,27 @@ pub async fn setup_environment(
     // Add provider to wallet
     wallet.set_provider(provider.clone());
 
-    // Deploy the target contract used for testing processing messages
-    let test_contract_id = Contract::deploy(
-        TEST_BRIDGE_FUNGIBLE_TOKEN_CONTRACT_BINARY,
-        &wallet,
-        TxParameters::default(),
-        StorageConfiguration::default(),
-    )
-    .await
-    .unwrap();
+    let test_contract_id = match configurables {
+        Some(c) => Contract::deploy_with_parameters(
+            TEST_BRIDGE_FUNGIBLE_TOKEN_CONTRACT_BINARY,
+            &wallet,
+            TxParameters::default(),
+            StorageConfiguration::default(),
+            c.into(),
+            Salt::default(),
+        )
+        .await
+        .unwrap(),
```

Comment:
```suggestion
        Some(config) => Contract::deploy_with_parameters(
            TEST_BRIDGE_FUNGIBLE_TOKEN_CONTRACT_BINARY,
            &wallet,
            TxParameters::default(),
            StorageConfiguration::default(),
            config.into(),
            Salt::default(),
        )
        .await
        .unwrap(),
```

I assume we cannot use `configurables` here

Reply:
no, reserved keyword.